qiskit/ignis/verification/randomized_benchmarking/fitters.py

```diff
@@ -35,7 +35,7 @@ class RBFitter:
     """
 
     def __init__(self, backend_result, cliff_lengths,
-                 rb_pattern=None):
+                 rb_pattern=None, is_interleaved=False):
```

Comment:
I think this can just be inferred by looking at the name

Reply:
Done